### PR TITLE
error handling in metadata exporter

### PR DESCRIPTION
**Pull request by hopkiw**

### CODE REVIEW

File: daisy_workflows/export_metadata/export-metadata.py [hunk 1]
```diff
@@ -90,14 +91,19 @@ def main():
     cmd = cmd_prefix + [package]
     process = None
     try:
-      process = subprocess.run(cmd, stdout=subprocess.PIPE, check=True)
+      process = subprocess.run(cmd, capture_output=True, check=True)
```
Comment (gaohannk) [inline daisy_workflows/export_metadata/export-metadata.py:94]: I found rhel default python3 is 3.6. And it doesn't have this argument. https://docs.python.org/3.6/library/subprocess.html#module-subprocess

Comment (hopkiw) [reply daisy_workflows/export_metadata/export-metadata.py:94]: i ran the workflow before submitting the PR :) 
default on the worker image is 3.7.3

**Pull request merged**